src/interrex_to_movebank.Rmd

```diff
@@ -31,7 +31,7 @@ metadata_colnames <- colnames(metadata)
 Set project:
 
 ```{r}
-project_id <- "HG_JUVENILE"
+project_id <- "ARMENIAN_GULL"
```

Comment:
You can commit this, but typically I don't change the project name, so it tends to switch each time you run an analysis.